rclrs/src/rcl_bindings.rs

```diff
@@ -4,5 +4,6 @@
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
 #![allow(clippy::all)]
+#![allow(missing_docs)]
```

Comment:
This should be fine, but I'm curious – why is this now needed?

Reply:
Because `rmw_request_id_t` is exposed to services, but the C struct is not documented.